pkg/internal/cli/status.go

```diff
@@ -134,6 +134,21 @@ func (s *Status) End(output Result) {
 	s.warningQueue = []string{}
 }
 
+func (s *Status) EndWithFailure(message string) {
+	s.QueueFailureMessage(message)
```

Comment:
```suggestion
func (s *Status) EndWithFailure(message string, a ...interface{}) {
	s.QueueFailureMessage(fmt.Sprintf(message, a...))
```

I see most calls to this function use `fmt.Sprintf` so it makes it more tidy to just use it internally